docs/src/snoopr.md

```diff
@@ -13,6 +13,14 @@ end
 ```
 
 Invalidations occur when there is a danger that new methods would supersede older methods in previously-compiled code.
+
+To record the invalidations caused by defining new methods, use the following syntax:
```

Comment:
Since you're providing a little more explanation about what's going on...technically, `@snoopr` does the recording, `invalidation_trees` does the analysis.

Reply:
Does it look better now?
https://github.com/timholy/SnoopCompile.jl/pull/110/commits/82973b03520aafe9e6ad89876cea900a4ef7f692